### PR TITLE
Ensure `any` as an IDL type always includes `null`.

### PR DESCRIPTION
**Pull request by joshualitt**

### CODE REVIEW

**Review by srujzs — APPROVED** [review]
LGTM, but just a few comments.

Food for thought: should we just typedef `JSAny` as `JSAny?` for `dart:js_interop`? dart2wasm and JS backends do not have the same semantics for nullability, and in general, if users were writing code that works on all backends, they'd probably want `?` unless they knew for sure they were excluding `null` and `undefined`.

IIRC, we were going to introduce an annotation to make semantics uniform, but I don't remember the status of that.

File: pubspec.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 name: web
 description: >-
   Lightweight DOM and JS bindings built around JS static interop.
-version: 0.1.1-beta
+version: 0.1.2-beta
```
[the remark this answers is not in the document]
Comment (kevmoo) [reply pubspec.yaml:4]: we already published 0.1.1-beta

[the remark this answers is not in the document]
Comment (srujzs) [reply pubspec.yaml:4]: Turns out we did! Resolving.

**Comment by srujzs** [comment]
Talked offline and we have consistent nullability semantics today across the backends, but the plan is to add an annotation for dart2wasm such that `null` and `undefined` roll into `JSAny`. This annotation will be explicit that it introduces inconsistent semantics across the JS backends, but that's opt-in so that's okay.

**Pull request merged**